README.md

```diff
@@ -149,7 +149,7 @@ Continuously move a servo connected to GPIO10 clockwise and anti-clockwise.
 ```js
 const Gpio = require('pigpio').Gpio;
 
-const motor = new Gpio(10, {mode: Gpio.OUTPUT});
+const motor = new Gpio(13, {mode: Gpio.OUTPUT});
```

Comment:
Why should the GPIO number specified here be changed from 10 to 13? In my opinion 10 is the correct GPIO number as it corresponds to the fritzing circuit diagram shown directly above the code [here](https://github.com/fivdi/pigpio/tree/v3.3.1#servo-control). Please not that the fritzing is for an older Pi 1 which has a 24 pin GPIO header, not a newer Pi with a 40 pin GPIO header.

Reply:
well, doesn't a Pi 1 seem a bit old and no longer too common - most users would no longer benefit from the sown circuit diagrams? I counted 4 Fritzing diagrams which I would volunteer to recreate for, e.g., a RasPi Zero. I should even have any required hardware at hand to actually test the examples in reality (incl. waveform generation which I could check with a pocket oscilloscope)